surveys/templates/partials/multiselect_geometry_widget.html

```diff
@@ -133,13 +134,13 @@
       if (layer.feature.properties.selected) {
         // Layer was already selected; turn it off and remove it from the list of
         // selected layers
+        resetHighlight(e);
```

Comment:
In particularly huge regions (e.g. Atlanta) the methods `SelectedLayers.pop()` and `SelectedLayers.push()` can have a slight lag, since the JavaScript needs to iterate the selectbox options and select/deselect options as appropriate. Because of this, I moved the functions that indicate UI changes (`resetHightlight()` and `layer.setStyle()` to the top of the block to make the user latency as low as possible.